services/storage/src/simcore_service_storage/api/v0/openapi.yaml

```diff
@@ -1,4 +1,4 @@
-openapi: 3.0.0
+openapi: 3.1.0
```

Comment:
is this thing fixed? last time this got upgraded then the swagger site was broken?